hw2/leo27945875/q2/test_q2.py

```diff
@@ -0,0 +1,7 @@
+import _vector
+import math
+
+
+def test_calc_angle():
+    angle = _vector.calc_angle(0., 1., 1., 0.)
+    assert abs(angle - 0.5 * math.pi) < 1e-5
```

Comment:
Could you please add the sufficient unittest according to the `validate.sh`?

https://github.com/yungyuc/nsdhw_24au/blob/3d417a41e674361630b8994f220952283ab046e9/hw2/validate.sh#L131-L138

Reply:
Hi, I have added more testcases.